packages/block-library/src/query-pagination-next/style.scss

```diff
@@ -0,0 +1,9 @@
+.wp-block-query-pagination-next-arrow {
+	margin-left: 2px;
```

Comment:
Maybe we should use one or two `ch` units here instead?

Reply:
Yeah, one `1ch` should scale very nicely.